### ISSUE TITLE
Add note about media sound through Zoom

### ISSUE THREAD

**Issue by heymenshan** [sircharlo/meeting-media-manager#3590]
### What happened?

When doing OBS integration, everything works perfectly, but the sound does not play automatically.

### To Reproduce

3. Do '....'
...

### What did you expect to happen?

I hope the sound comes out.

### Version

v24.11.5

### What platform are you seeing the problem on?

macOS

### Confirmations

- [X] I confirm that I have read the documentation website.
- [X] I confirm that I have searched through the existing issues to make sure my bug report is not a duplicate.
- [X] I confirm that I have searched through the existing discussions to make sure my bug report is not a duplicate.

**Comment by ux-git** [comment]
Do you mean that there's no sound while playing videos from Meeting Media Manager? The sound is not directly sent to OBS, as the video stream works like a virtual camera with no sound. To have the music/video sound available in Zoom automatically, make sure to [turn on the Original Audio ON](https://github.com/sircharlo/meeting-media-manager/discussions/1599#discussioncomment-5157328) every time before the meeting starts.

@mtdvlpr I think it definitely worth mentioning in the documentation. Either along with the OBS integration or at least in the FAQs, or both but with different phrasing. For example something like this for FAQ: 
> Does Meeting Media Manager automatically sends the media audio to Zoom?

And for the Using at a Kingdom Hall, to describe in more details with clear step-by-step where and how to enable the Original audio AND to enable it every time on new Zoom startup.

**Comment by e-Feitosa** [comment]
> E para o Usando em um Salão do Reino, para descrever com mais detalhes com passo a passo claro onde e como habilitar o áudio original E habilitá-lo sempre na nova inicialização do Zoom.

![image](https://github.com/user-attachments/assets/a70f7714-bee1-4c8a-9d91-412ef5bf51bb)

**Comment by ux-git** [comment]
@e-Feitosa yes, that's also a good alternative

**Comment by e-Feitosa** [comment]
@ux-git, Taking advantage of this topic, in the previous instructions on using the M3, a tip was given to add a command to the OBS Studio shortcut to activate the camera when opening OBS Studio.

My question is: **Does anyone know something similar for Zoom meetings that could be done in the Zoom meeting shortcut to automatically enable the option to share computer sound?** 

The current method is: 
**Default Settings:** _Before starting the meeting, go to sharing, then click on the advanced tab in Zoom and enable the option to share computer sound. This can be done before starting a meeting, but it has to be done every time you start Zoom._

However, I know it might be possible to minimize this, the options below indicate this, but I don't have the technical knowledge to do it:

**Macros or Scripts:** You can create a script that automates opening Zoom and enabling sound sharing. Tools like AutoHotkey (for Windows) or Automator (for Mac) can be useful.

**Plugins or Extensions:** [I haven't found anything so far] There are plugins or extensions that can add this functionality to Zoom. Sometimes third-party developers create solutions for these specific needs. 

**Note**: _On most computers, especially laptops, even using an audio interface and following the recommended configuration, the sound is very noisy or doesn't come through on Zoom, hence the need to share computer audio._

**Comment by ux-git** [comment]
> However, I know it might be possible to minimize this, the options below indicate this, but I don't have the technical knowledge to do it:

We're using the Original Audio setting. You have to turn it on only once, and then before every meeting click the "Original audio" button which is one click away. This scenario minimizes the click count from 4 to 1, but still has to be done before every meeting. Forgetting it will result in bad media audio in Zoom.

**Comment by e-Feitosa** [comment]
> We're using the Original Audio setting. (....) 

Look how interesting, in my previous congregation, we used a Desktop with the audio option set to automatic, and it worked fine without any problems. This year, I was assigned to another congregation, and here we use a laptop. I currently use the same option, and I also have no problems; the audio is perfect. Why did I ask the above? Here in the region, I support congregations that need help, and practically all of them need to share the computer's audio. I tried using the Original Sound option for music, with the three options checked or unchecked and with the original audio option active. In some, it didn't work or presented noise for those on Zoom. The only way to solve it was to share the computer's audio in the Zoom meeting every time it connects to the meeting.

**Comment by mtdvlpr** [comment]
@sircharlo, what do you think about adding some explanation about this to the docs?

**Comment by ux-git** [comment]
> strictly speaking, it does not belong in the M³ documentation.

You're totally right, but that's exactly why FAQs and documentation exist – to explain how the app works, and provide the existing good workarounds to help them set everything up as fast as possible and as good as possible. All this while still letting users to try it out or reconfigure themselves.

> ...Nor do we take the view: ‘No one trained me! Let him learn on his own.'

😄

**Comment by sircharlo** [comment]
I can't see how adding a section to the docs could hurt. I'd probably lay out a few options with the caveat that troubleshooting that falls outside the scope of the app.

**Comment by e-Feitosa** [comment]
@sircharlo, The understanding that this audio issue is not related to the M3 is correct. A note explaining this could be very helpful, especially because most people, when sharing their screen on Zoom, select both options (audio and optimize). When they install the M3, some think they don't need to do anything else regarding the audio, which leads to audio problems on Zoom, and they think it's the M3. I believe a note explaining this could help minimize this problem, as many end up thinking they don't need to configure anything else besides installing the M3

**Comment by sircharlo** [comment]
Bump

**Comment by sircharlo** [comment]
Bump

**Comment by mtdvlpr** [comment]
Bump

**Comment by mtdvlpr** [comment]
Bump

**Comment by sircharlo** [comment]
Thanks for these suggestions! Take a look at the updated docs and let me know what you think! 😄
